src/js/components/Select/doc.js

```diff
@@ -23,6 +23,27 @@ export const doc = Select => {
       { active, disabled, selected } keys indicating the current state
       of the option.`,
     ),
+    clear: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        position: PropTypes.oneOf(['top', 'bottom'])
+          .description(
+            `Add a clear value to the top or at the bottom of the
```

Comment:
I think we should use the term `option` instead of `value` or `button` in our descriptions. I think that aligns a bit better with `options` being what we show in the drop.

Reply:
changes made